packages/icons-workflow/bin/build.js

```diff
@@ -38,7 +38,8 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */`;
 
-const S1IConsPackageDir = '@adobe/spectrum-css-workflow-icons/dist/18';
+const S1IConsPackageDir =
+    '@adobe/spectrum-css-workflow-icons/dist/spectrum-css-workflow-icons/dist/18';
```

Comment:
```suggestion
    '@adobe/spectrum-css-workflow-icons/dist/18';
```

This doesn't look like the correct path - why are there nested dist folders?

Reply:
Needed to modify this to bring in data from v 1.6.0